### ISSUE TITLE
Adding "nil" to bibliography entry

### ISSUE THREAD

**Issue by HiramTheHero** [zepinglee/citeproc-lua#63]
**Describe the bug**
The word "nil" is added to a bibliography entry after "U.S.C" when citing legislation.

**Additional information**
- TeX distribution: TeX Live 2024 - MacOS
- Package `citation-style-language` version: v0.4.7
- LaTeX engine: `lualatex`

**To Reproduce**
```tex
\documentclass{article}

\begin{filecontents}[overwrite, noheader]{\jobname.json}
[
	{
		"id": "AmericaCOMPETESAct2010",
		"type": "legislation",
		"abstract": "To invest in innovation through research and development, and to improve the competitiveness of the United States.",
		"chapter-number": "111",
		"container-title": "U.S.C",
		"language": "English (United States)",
		"number": "111-358",
		"section": "525",
		"title": "America Creating Opportunities to Meaningfully Promote Excellence in Technology, Education, and Science Reauthorization Act of 2010",
		"URL": "https://www.govinfo.gov/content/pkg/PLAW-111publ358/html/PLAW-111publ358.htm",
		"volume": "42",
		"author": [
			{
				"literal": "111th Congress"
			}
		],
		"issued": {
			"date-parts": [
				[
					"2011"
				]
			]
		}
	}
]

\end{filecontents}

\usepackage[style = apa]{citation-style-language}
\addbibresource{\jobname.json}
\usepackage{hyperref}

\begin{document}
\cite{AmericaCOMPETESAct2010}
\printbibliography
\end{document}
```

**Screenshots**
<img width="864" alt="Screen Shot 2024-05-24 at 3 58 32 PM" src="https://github.com/zepinglee/citeproc-lua/assets/22198719/ce8c278b-af8c-4c8b-bcc2-d6c88594a1d4">

**Comment by zepinglee** [comment]
This bug is already fixed in latest v0.4.9.

<https://github.com/zepinglee/citeproc-lua/blob/e3eb0d52b9bf2d8f4a42a29c0fa03dcacc3151e5/tests/latex/luatex-1/issue-63.tlg#L18>

**Comment by HiramTheHero** [comment]
Thank you for the information. After building the latest version of citeproc-lua and using it, the error went away.
